less/navigation.less

```diff
@@ -1 +1,41 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+.nav-bar {
+    max-width: 1000px;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    margin: 10px auto 0 auto;
+    padding-bottom: 20px;
+    justify-content: space-around;
+    border-bottom: 2px dashed @navigation-border;
+    nav {
+        // display: flex;
```

Comment:
i would use display: flex here and then justify-content: space-evenly, and finally a width of ~40-50% which will cause your nav to take up that much of the whole header while staying off to one side due to the space-between on nav-bar
then I would change that width to 100% in a media query for mobile